### ISSUE TITLE
Issues with alpine / armv7 libuv buildbots

### ISSUE THREAD

**Issue by santigimeno** [nodejs/build#1186]
Lately we've been having issues with the alpine and armv7 libuv bots due to some missing packages

- alpine [error](https://ci.nodejs.org/job/libuv-test-commit-linux/799/nodes=alpine-last-latest-x64/console):
```
autogen.sh: line 28: automake: not found
sh: out of range
sh: out of range
+ libtoolize --copy
autogen.sh: line 1: libtoolize: not found
Build step 'Execute shell' marked build as failure
Notifying upstream projects of job completion
Finished: FAILURE
```

- arm7 [error](https://ci.nodejs.org/job/libuv-test-commit-linux/799/nodes=armv7-wheezy/console): 
```
+ libtoolize --copy
autogen.sh: 43: autogen.sh: libtoolize: not found
Build step 'Execute shell' marked build as failure
Notifying upstream projects of job completion
Finished: FAILURE
```

Having this fixed would be great. Thanks!

**Comment by rvagg** [comment]
sorry @santigimeno, I'll try and get this fixed this week

**Comment by bnoordhuis** [comment]
It's mid May and I regret to say it's still an issue.

**Comment by maclover7** [comment]
Taking a look -- looks like the ARM v7 problem has been fixed, looking at getting Alpine squared away

**Comment by maclover7** [comment]
Fix is in #1287
